scripts/classifier_prf/cross_validate.py

```diff
@@ -1,29 +1,45 @@
-import pandas as pd
+import argparse
 import json
 import os
-import argparse
+import pandas as pd
+
+
+def get_file_extension(rm3: bool):
+    return '_bm25+rm3.txt' if rm3 is True else '_bm25.txt'
 
 
-def read_topics_alpha_map(anserini_root, collection, run_file, classifier, rm3):
+def get_file_path(run_file, collection, classifier, alpha: str, rm3: bool):
+    res = f'{run_file}/{collection}/{collection}_{classifier}_A' + alpha
+    return res + get_file_extension(rm3)
+
+
+def get_res_file_path(run_file, collection, classifier, alpha: str, rm3: bool):
+    res = f'{run_file}/cv/{collection}/scores_{collection}_{classifier}_A' + alpha
+    return res + get_file_extension(rm3)
+
+
+def read_topics_alpha_map(anserini_root, collection, run_file, classifier, rm3: bool):
     res_paths = []
+
     for num in range(0, 11):
         alpha = str(num / 10)
-        if rm3:
-            file_path = f'{run_file}/{collection}/{collection}_{classifier}_A' + alpha + '_bm25+rm3.txt'
-        else:
-            file_path = f'{run_file}/{collection}/{collection}_{classifier}_A' + alpha + '_bm25.txt'
-        res_filename = f'{run_file}/cv/{collection}/scores_{collection}_{classifier}_A' + alpha + '_bm25.txt'
+        file_path = get_file_path(run_file, collection, classifier, alpha, rm3)
+        res_filename = get_res_file_path(
+            run_file, collection, classifier, alpha, rm3)
+
         res_paths.append(res_filename)
-        cmd = f'{anserini_root}/eval/trec_eval.9.0.4/trec_eval -q -m map -m P.30 {anserini_root}/src/main/resources/topics-and-qrels/qrels.core18.txt {file_path} > {res_filename}'
+        cmd = f'{anserini_root}/eval/trec_eval.9.0.4/trec_eval -q -m map -m P.30 {anserini_root}/src/main/resources/topics-and-qrels/qrels.{collection}.txt {file_path} > {res_filename}'
```

Comment:
Thanks for fixing the bug. This line is a little too long to read. Can you change it to two lines?

Reply:
Will do!